src/reference.yaml

```diff
@@ -16131,6 +16136,8 @@ definitions:
         '2021-01-01': New Year's Day
         '2021-01-02': No trading on Saturday
         '2021-01-03': No trading on Sunday
+      offline_conversion_dates:
+        - '2020-11-23'
```

Comment:
The response should be in Array `- ['2020-11-23']`

the same as in https://github.com/ccycloud/currencycloud-swagger-private/pull/177

Reply:
This is the correct way of defining an array response in the yaml

Reply:
Got you, thanks.

Updated as well in https://github.com/ccycloud/currencycloud-swagger-private/pull/177